server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/ACVRDownloadByCounty.java

```diff
@@ -50,7 +55,7 @@ public EndpointType endpointType() {
    */
   @Override
   public String endpointName() {
-    return "/acvr/:counties";
+    return "/acvr/county/:counties";
```

Comment:
I did this, but it will still need a bit of work later - right now it's still county names, but if we're going to send county ID numbers around instead (i.e., when you upload a file you specify the county that's uploading it by ID number, or we know because they're logged in what their ID is), that's an easy change later.